src/components/schedule-visualizer/robot-trajectory.tsx

```diff
@@ -19,51 +20,44 @@ export const RobotTrajectory = React.forwardRef(function(
   const { trajectory, conflicts, footprint, ...otherProps } = props;
   const theme = useTheme();
 
-  const pathD = React.useMemo(() => {
-    const knots = rawKnotsToKnots(trajectory.segments);
-    const coeff = knotsToSegmentCoefficientsArray(knots);
-    const bezierSplines = coeff.map(bezierControlPoints);
-
-    let d = `M ${bezierSplines[0][0][0]} ${-bezierSplines[0][0][1]} C `;
-    bezierSplines.map(
-      bzCurves =>
-        (d +=
-          `${bzCurves[1][0]} ${-bzCurves[1][1]} ` +
-          `${bzCurves[2][0]} ${-bzCurves[2][1]} ` +
-          `${bzCurves[3][0]} ${-bzCurves[3][1]} `),
-    );
-
-    return d;
-  }, [trajectory]);
-
-  const color = React.useMemo(
+  const pathColor = React.useMemo(
     () =>
-      conflicts.includes(trajectory.id) ? theme.palette.error.main : theme.palette.success.main,
+      conflicts.includes(trajectory.id)
+        ? theme.palette.error.main
+        : ColorManager.getPathColor(trajectory.id),
```

Comment:
If we are using a fixed color for the conflicting trajs and a calculated color for non-conflicting trajectories we should take care that the non-conflict color does not end up being too similar to the conflicting color.

Reply:
Makes sense. Will do. Thanks

Reply:
Done!